documentation/ame-guide/modules/ROOT/pages/namespace/import-package.adoc

```diff
@@ -3,16 +3,16 @@
 
 To import a package with namespaces, proceed as follows:
 
-. Click *File* ->  *Import Package* in the xref:getting-started/ui-overview.adoc#menu-file[file menu].
+. Click the xref:getting-started/ui-overview.adoc#menu-file[File menu] and select *Import Package*.
 +
-→ An Explorer window opens.
+→ A File Explorer window opens.
```

Comment:
we do not use the arrow everywhere right? maybe we need to double-check

Reply:
For results of tasks, so in the last line of this description it is currently missing.